### PR TITLE
Change the Mnemonic from Availability to Unavailability

### PR DESCRIPTION
**Pull request by dantebarbieri**

### CODE REVIEW

**Comment by dantebarbieri** [comment]
If the lambdas look good, please redeploy every single one that changed. This is almost every single lambda.

**Comment by dantebarbieri** [comment]
The request to redeploy was directed at @Shane-McGookey

File: scripts/generate_schedulinginfo.js [hunk 1]
```diff
@@ -84,105 +128,71 @@ client
...
+                  intersects = true
+                  break
+                }
+              }
+              if (intersects) break
+            }
+            if (!intersects) {
+              person_courses.push(section)
+            } else {
+              --i; // bad, very very bad
```
Comment (benbeauchamp7) [inline scripts/generate_schedulinginfo.js:173]: ?

**Review by benbeauchamp7 — APPROVED** [review]
lgtm

**Pull request merged**